lmfdb/templates/workshops.html

```diff
@@ -32,6 +32,20 @@
 
 <br>
 
+<li>
+Mod ell modular forms, <br>
+August 5-9, 2019,
+<a href="https://aimath.org/">American Institute of
```

Comment:
Warwick Mathematics Institute https://warwick.ac.uk/fac/sci/maths/

Reply:
Sorry, got confused by the AIM SQuaRE